pkg/minishift/registration/utils.go

```diff
@@ -93,7 +93,7 @@ func UnregisterHostVM(host *host.Host, param *RegistrationParameters) (bool, err
 	}
 
 	return doRegistrationHostAction(
-		"Unregistering machine",
+		"Unregistering machine\n",
```

Comment:
isn't this needed for all the registration host actions? if soo, make that a println